src/main/resources/jupyter/init-actions.sh

```diff
@@ -194,6 +194,8 @@ if [[ "${ROLE}" == 'Master' ]]; then
       COMPOSE_FILES+=(-f /etc/`basename ${RSTUDIO_DOCKER_COMPOSE}`)
     fi
 
+    cat "${COMPOSE_FILES[@]}"
```

Comment:
@vkumra-broad should probably be `echo` rather than `cat` - seeing cluster create failures in dev:

```
+ '[' '!' -z us.gcr.io/broad-dsp-gcr-public/leonardo-jupyter:acdb39f5f1a1 ']'
+ COMPOSE_FILES+=(-f /etc/`basename ${JUPYTER_DOCKER_COMPOSE}`)
++ basename gs://leoinit-all-of-us-a02a5feb-265f-45ab-bbdc-519cbefea4d7/jupyter-docker-compose.yaml
+ '[' '!' -z ']'
+ cat -f /etc/proxy-docker-compose.yaml -f /etc/jupyter-docker-compose.yaml
cat: invalid option -- 'f'
Try 'cat --help' for more information.
```